examples/use-module-defaults-group.yml

```diff
@@ -11,17 +11,17 @@
     - name: Create a volume
       hetzner.hcloud.volume:
         name: my-volume
-        location: fsn1
+        location: hel1
         size: 100
         state: present
       register: volume
 
     - name: Create a server
       hetzner.hcloud.server:
         name: my-server
-        server_type: cx22
+        server_type: cpx22
         image: debian-12
-        location: fsn1
+        location: hel1
```

Comment:
Based on the web console cpx22 is available in both fsn1 and hel1 so the swap of location would be better in its own commit with separate rationale in the commit message.